packages/zoe/src/contractFacet/zcfZygote.js

```diff
@@ -237,20 +231,35 @@ export const makeZCFZygote = async (
     },
     getInstance: () => getInstanceRecord().instance,
     setOfferFilter: strings => E(zoeInstanceAdmin).setOfferFilter(strings),
+    getOfferFilter: () => E(zoeInstanceAdmin).getOfferFilter(),
+  });
+
+  const SeatDataShape = harden({
+    proposal: ProposalShape,
+    initialAllocation: AmountKeywordRecordShape,
+    seatHandle: SeatShape,
+  });
+
+  const HandleOfferObjShape = M.interface('HandleOffer object', {
```

Comment:
This is an interface guard, not a pattern. We reserve the name `*Shape` for patterns.

You do need both. Separately, can we drop `Obj` from the name? I know we added it to distinguish it from the function case, but object should be the understood default for remotables.

```suggestion
  const HandleOfferShape = M.remotable('HandleOffer');
  const HandleOfferInterface = M.interface('HandleOffer', {
```

In this file, you only need `HandleOfferShape`, below. The purpose of `HandleOfferInterface` is for durable contracts to import, in order to create FarClasses or FarInstances. durable-coveredCall.js doesn't yet need it because still uses `vivifyKind`. So you should move `HandleOfferInterface` somewhere it can be exported, for contracts to import it.

Separately, `M.remotable()` does take an optional string `label` argument whose only purpose is better error messages. My convention is to use the interface name. My convention for interface names is to use an identifier than names what the corresponding interface is about.

Reply:
done